lbryio/lbry-sdk
# Find unused ports for media server

Fixes #3032 and other duplicate or similar bugs. If the default 5280 port is taken try next 10 ports and store the open port back in the config class as I've seen that some other classes are reading from it.
Looking forward to your feedback, thank you.
Testing:
- Run `nc -klv 5280` to start listening on port 5280
- Run `lbrynet start` to start lbrynet
- Notice the logs:
```
2021-01-14 15:06:30,775 INFO     lbry.extras.daemon.daemon:517: Address localhost:5280 is already in use, trying next port
2021-01-14 15:06:30,776 INFO     lbry.extras.daemon.daemon:526: media server listening on TCP ::1:5281
```
